TaylorSource.xcodeproj/project.pbxproj

```diff
@@ -219,6 +234,18 @@
 			name = Frameworks;
 			sourceTree = "<group>";
 		};
+		C44F8F4D1CF99A0E008A8478 /* CoreData */ = {
+			isa = PBXGroup;
+			children = (
+				C44F8F4E1CF99A0E008A8478 /* NSFRCDatasource.swift */,
+				C44F8F4F1CF99A0E008A8478 /* NSFRCFactory.swift */,
+				C44F8F501CF99A0E008A8478 /* NSFRCIndexedUpdates.swift */,
+				C44F8F511CF99A0E008A8478 /* NSFRCUpdateHandler.swift */,
+			);
+			name = CoreData;
+			path = Sources/CoreData;
+			sourceTree = SOURCE_ROOT;
+		};
```

Comment:
These files were already present in the repo, but not included in the project. I just added them for completeness.